File: Models/QuestionAnswer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EducationalSoftwareServer.Models
{
    public class QuestionAnswer
    {
        public int QuestionAnswerId { get; set; }
        public int QuestionId { get; set; }
        public string Answer { get; set; }
        public bool IsRight { get; set; }

    }
}

File: Client/EducationalSoftwareClient/src/app/models/questionAnswer.ts

export class QuestionAnswer {
    questionAnswerId: number;
    questionId: number;
    answer: string;
    isRight: boolean;

    constructor(){
        this. questionAnswerId = 0;
        this. questionId = 0;
        this.answer = "";
        this.isRight = false;
    }
}